### ISSUE TITLE
Introduce geographical hierarchy in cities

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#282]
http://world.openfoodfacts.org/cities

Use content within parenthesis to introduce a geographical hierarchy
Rezé (Loire-Atlantique, France) >> Loire-Atlantique >> France >> European Union

**Comment by hangy** [comment]
Maybe link cities to Wikidata instead and do hierarchies through [P131](https://www.wikidata.org/wiki/Property:P131)?

**Comment by teolemon** [comment]
Also, shouldn't German cities appear now ?

**Comment by hangy** [comment]
No, the city code is exclusive to France right now. (See `Tags::get_city_code` and code after `# Load cities for emb codes`.)
